3. Installation/4. Manual Installation/Multiple Instances to Improve Performance/README.md

```diff
@@ -21,34 +21,69 @@ We'll be working with Nginx in our examples, but it should be possible with othe
 
 ## Run multiple instances of Rocket.Chat
 
-We'll assume that you've configured Rocket.Chat to run as a service. Since we want to run multiple instances
-simultaneously, we need to run two services. The only difference is the service name, port, and logging path.
-If you don't have a service yet, the easiest way to do this for Rocket.Chat is to use `forever-service` to
-create the services and run them with `forever` for you.
-
-Let's create two services: `rocketchat1` runs on port 3001 and `rocketchat2` runs on port 3002.
-
-    sudo npm install -g forever
-    sudo npm install -g forever-service
-    cd ~/Rocket.Chat
-    sudo forever-service install -s main.js -e "ROOT_URL=https://example.com/ MONGO_URL=mongodb://localhost:27017/rocketchat MONGO_OPLOG_URL=mongodb://localhost:27017/local PORT=3001" rocketchat1
-    sudo forever-service install -s main.js -e "ROOT_URL=https://example.com/ MONGO_URL=mongodb://localhost:27017/rocketchat MONGO_OPLOG_URL=mongodb://localhost:27017/local PORT=3002" rocketchat2
-
-On debian/ubuntu, these generate upstart scripts in `/etc/init/rocketchat1.conf` and `/etc/init/rocketchat2.conf`.
-Notice that it automatically uses the service name to configure logging, e.g., `/var/log/rocketchat1.conf`.
+We'll assume that you've configured Rocket.Chat to run as a systemd service. Since we want to run multiple instances
+simultaneously, we need to run at least two services. The only difference is the service name and port.
+If you don't have a service yet, the easiest way to do this for Rocket.Chat is to create a file in /usr/lib/systemd/system/
+and call it rocketchat.service
+```
+[Unit]
+Description=Rocket.Chat Server
+After=syslog.target
+After=network.target
+
+[Service]
+Type=simple
+Restart=always
+StandardOutput=syslog
+SyslogIdentifier=RocketChat
+User=rocketchat
+Group=rocketchat
+Environment=MONGO_URL=mongodb://your_mongodb:27017/your_replica_set_name
+Environment=MONGO_OPLOG_URL=mongodb://your_mongodb1:27017local?replicaSet=your_replica_set_name
+Environment=ROOT_URL=https://your_rocketchat_domain.com
+Environment=PORT=3000
+WorkingDirectory=/path.to.rocketchat/rocket.chat
+ExecStart=/usr/local/n/versions/node/4.8.4/bin/node /path.to.rocketchat/rocket.chat/bundle/main.js
```

Comment:
Hard coding to specific version isn't good.  Especially considering next version of Rocket.Chat will be using new version of nodejs